setup.py

```diff
@@ -10,19 +10,18 @@
     "click>=7.0",
     "rasterio>=1.1",
     "numpy~=1.15",
-    "morecantile>=3.0,<3.1",
+    "morecantile>=3.1,<4.0",
```

Comment:
I think it's safe to allow morecantile up to 4.0

Reply:
I think it's ok to limit this new version of rio-cogeo of only using morecantile>=3.1 because the change of the WebMercatorDefinition.